documentation/c01-whatis.sil

```diff
@@ -82,11 +82,105 @@ SILE does not benefit from the large ecosystem and community that has grown up a
+Its math support was barely at a proof-of-concept stage and definitely not robust enough for scientific publishing work.
+This niche was exactly the felt need of Typst’s developers; the major focus right out of the gate was math and science papers.
+In the first releases it was more than a little bit rough in the prose department.
+Since then SILE’s math chops have greatly expanded to cover the majority of MathML.
+Typst’s feature set for handling prose have also improved.
+That being said the initial focus and energy of each project is still very visible.
+
+\begin{itemize}
+
+\item{\strong{Field of focus:}
+	SILE was initially conceived to handle hard problems in automated typesetting of books in many world orthographies.
+	Typst has some multi-lingual support but does not have robust support for the wide variety of typesetting issues posed by the world’s scripts and languages to the extent SILE does.
+	Typst was initially conceived for work in the academic sciences, and the robustness of the math and figure APIs directly reflects this.
+	SILE has math support but it has not been exercised to nearly the extent of Typst’s.}
+
+\item{\strong{Input formats:}
+	SILE was conceived to handle multiple input formats; out of the box it supports a LaTeX-like syntax but also includes an XML reader suitable for inputting other formats.
+	In addition writing input drivers is relatively easy and 3rd party ones for Markdown, DJot, and others have been developed.
+	Typst on the other hand developed their own input format, loosely based on Markdown but distinct and unique; it does not support input in anything else.
+	Pandoc support allows some other formats to be converted and imported, but not worked with natively.}
+
+\item{\strong{Scripting:}
+	SILE exposes (and indeed is actually mostly implemented in) Lua, an industry standard embedded scripting language with an already large ecosystem of code and packages.
+	Lua’s syntax and features are well known and documented.
+	Besides being used in other Typesetting applications\footnote{%
+		SILE was initially implemented in JavaScript, then ported to Lua.
+		The thought behind the switch to Lua was to gain advantage of its use in LuaLaTeX and other typesetting applications.
+		In practice, using the same language as other typesetting systems has not really provided much of the hoped for opportunities for code re-use.
+		It turns out algorithms and expertise themselves are more relevant than the language they happen to be written in and LuaLaTeX specific libraries are not very flexible.}
+	it can be found embedded in web servers, game engines, code editors, and a myriad of other applications.
+	Typst’s approach to scripting was to invent their own language from scratch.
+	The language they invented is clean, logical, and robust—but also unique and, to some extent, constraining.}
+
+\item{\strong{Security model:}
+	Typst attempts to make the renderer safe to run on arbitrary input.
```

Comment:
"attempts to": does it imply it doesn't succeed? Or that are limitations? The wording is weird.